### PR TITLE
创建项目框架

### PR DESCRIPTION
**Pull request by dongjiajie**
## 创建一个helloworld文件

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (CNBlackJ) [inline .idea/vcs.xml]: It should be deleted!!
@dongjiajie

File: assets/js/app/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by dongjj on 2017/7/11.
```
Comment (CNBlackJ) [inline assets/js/app/index.js]: I do not like this comment.

File: assets/js/app/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ */
+var koa = require('koa');
```
Comment (CNBlackJ) [inline assets/js/app/index.js]: Since ES6 we should using `let` and `const` to instead of `var`.
@dongjiajie

**Comment by dongjiajie** [comment]
#4 
和这个的pull request 重复

**Pull request closed without merging**